components/team_card/team_card_storage.lua

```diff
@@ -0,0 +1,92 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:TeamCard/Storage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Custom = require('Module:TeamCard/Custom')
+local String = require('Module:StringUtils')
+-- TODO: Once the Template calls are not needed (when RL has been moved to Module), deprecate Qualifier
+local Qualifier = require('Module:TeamCard/Qualifier')
+local Variables = require('Module:Variables')
+
+local TeamCardStorage = {}
+
+function TeamCardStorage.saveToLpdb(args, teamObject, players, playerPrize)
+	local team, teamTemplate
+	local image, imageDark
+
+	if type(teamObject) == 'table' then
+		if teamObject.team2 or teamObject.team3 then
+			team = 'TBD'
+		else
+			teamTemplate = teamObject.teamtemplate
```

Comment:
Isn't this case a table too?

Reply:
Updated (local) variable name to reflect that it's the TTName and not the TTObject.

Reply:
My question wasn't answered though, isn't `teamObject` still a table?

Reply:
Yes?

Reply:
Sorry, misread